yapapi/executor/subtask.py

```diff
@@ -82,58 +82,58 @@ def _stop(self, retry: bool = False):
 
     @staticmethod
     def for_handle(
-        handle: Handle["Task[TaskData, TaskResult]"],
-        queue: SmartQueue["Task[TaskData, TaskResult]"],
+        handle: Handle["SubTask[Data, Result]"],
+        queue: SmartQueue["SubTask[Data, Result]"],
         emitter: Callable[[events.Event], None],
-    ) -> "Task[TaskData, TaskResult]":
-        task = handle.data
-        task._handle = (handle, queue)
-        task._start(emitter)
-        return task
+    ) -> "SubTask[Data, Result]":
+        subtask = handle.data
+        subtask._handle = (handle, queue)
+        subtask._start(emitter)
+        return subtask
 
     @property
-    def data(self) -> TaskData:
+    def data(self) -> Data:
         return self._data
 
     @property
-    def output(self) -> Optional[TaskResult]:
+    def output(self) -> Optional[Result]:
         return self._result
 
     @property
     def expires(self) -> Optional[datetime]:
         return self._expires
 
-    def accept_task(self, result: Optional[TaskResult] = None) -> None:
-        """Accept task that was completed.
+    def accept_result(self, result: Optional[Result] = None) -> None:
+        """Accept subtask that was completed.
 
-        Must be called when the results of a task are correct.
+        Must be called when the results of the subtask are correct.
 
         :param result: computation result (optional)
         :return: None
         """
         if self._emit:
-            self._emit(events.TaskAccepted(task_id=self.id, result=result))
-        assert self._status == TaskStatus.RUNNING, "Accepted task not running"
-        self._status = TaskStatus.ACCEPTED
+            self._emit(events.SubTaskAccepted(subtask_id=self.id, result=result))
+        assert self._status == SubTaskStatus.RUNNING, "Accepted subtask not running"
+        self._status = SubTaskStatus.ACCEPTED
         self._result = result
         self._stop()
         for cb in self._callbacks:
-            cb(self, TaskStatus.ACCEPTED)
+            cb(self, SubTaskStatus.ACCEPTED)
 
-    def reject_task(self, reason: Optional[str] = None, retry: bool = False) -> None:
-        """Reject task.
+    def reject_result(self, reason: Optional[str] = None, retry: bool = False) -> None:
+        """Reject subtask.
```

Comment:
again, it's the _result_ of an execution job that's rejected not the "subtask" - or fragment...